media/js/src/assetDetail/AssetDetail.jsx

```diff
@@ -288,6 +288,39 @@ export default class AssetDetail extends React.Component {
 
                     <div className="col-sm-6">
                         {media}
+                        {this.state.tab === 'viewItem' && (
```

Comment:
I'd like to strongly lobby for this section to be placed under the "View the Original Item" tab. It feels unmoored below the video. I'll merge this, and we can discuss.